ssr/src/page/post_view/video_loader.rs

```diff
@@ -43,6 +54,15 @@ pub fn BgView(
         Some(())
     });
 
+    create_effect(move |_| {
+        if is_eligible_onboarding_post.get() && (!is_onboarded.get()) {
+            set_show_onboarding_popup.update(|show| *show = true);
+        } else {
+            set_show_onboarding_popup.update(|show| *show = false);
+        }
+        Some(())
```

Comment:
nit: do you need this line?

Reply:
Yes it is required , When user gets onboarded on the app either by skipping or completing tutorial, i'm changing the local storage to true, and then updating set_show_onboarding_popup here accordingly.

Reply:
I am talking about line 63 specificially

Reply:
Nope, cleaned it  in the last commit.